ftests/core/src/main/java/org/commonjava/indy/ftest/core/content/prefetch/RemotePrefetchRescanTest.java

```diff
@@ -165,7 +165,7 @@ public void run()
         assertThat( fileJar2.exists(), equalTo( false ) );
         assertThat( fileSrc2.exists(), equalTo( false ) );
 
-        Thread.sleep( 1500 );
+        waitForEventPropagation();
```

Comment:
This sleeping depends on the rescan time interval, so it could not use the same one here and need to adjust according to that interval.